tests/js/client/shell/shell-admin-cluster-rebalance.js

```diff
@@ -338,6 +335,75 @@ function clusterRebalanceOtherOptionsSuite() {
   };
 }
 
+function clusterRebalanceWithMovesToMakeSuite() {
+  let prevDB = null;
+  const cn = "col1";
+
+  return {
+
+    setUpAll: function() {
+      prevDB = db._name();
+      db._createDatabase(database);
+      db._useDatabase(database);
+    },
+
+    tearDownAll: function() {
+      db._useDatabase(prevDB);
+      db._dropDatabase(database);
+    },
+
+
+    testCalcRebalanceAfterUnbalanced: function() {
+      for (let i = 1; i <= 3; ++i) {
+        const toServer = "DBServer000" + i;
+        db._create(cn, {numberOfShards: 10, replicationFactor: 1});
+        const plan = arango.GET("/_admin/cluster/shardDistribution").results[cn].Plan;
```

Comment:
Everything from here on should be wrapped in a `try...finally`, so that dropping the collection at the end will always be executed.